src/packages/@ncigdc/components/Modals/GroupValues/index.tsx

```diff
@@ -163,12 +131,11 @@ const GroupValuesModal = ({
                 }}
                 style={visualizingButton}
                 />
-
               <Button
-                leftIcon={<Group style={{ width: '10px' }} />}
                 disabled={Object.values(selectedGroupBins).filter(Boolean).length < 2}
+                leftIcon={<Group style={{ width: '10px' }} />}
```

Comment:
we ought to refactor this component to not take the rendered component as a prop... later